community/rust/APKBUILD

```
@@ -6,8 +6,8 @@
 pkgname=rust
 pkgver=1.36.0
 _llvmver=8
-_bootver=1.35.0
-pkgrel=0
+_bootver=1.36.0
+pkgrel=1
 pkgdesc="The Rust Programming Language"
 url="https://www.rust-lang.org"
 arch="x86_64 armv7 armhf"
```

```suggestion
arch="x86_64 aarch64 armv7 armhf"
```
We need the builder to not exit immediately.

---

Well, I'm currently working on that but until then we'll need to have aarch64 disabled.